os/arch/arm/src/s5j/s5j_i2s.c

```diff
@@ -1650,9 +1650,14 @@ static int i2s_stop(struct i2s_dev_s *dev)
 	struct s5j_i2s_s *priv = (struct s5j_i2s_s *)dev;
 	irqstate_t flags;
 	struct s5j_buffer_s *bfcontainer;
+	volatile u32 reg;
 	DEBUGASSERT(priv);
```

Comment:
added this but I cant see the usage of this anywhere

Reply:
It is used in line num 1717.